### PR TITLE
Issue #SB-0000 fix: Update to Angular 11 from 10

### PR DESCRIPTION
**Pull request by princegupta1131**

### CODE REVIEW

**Comment by princegupta1131** [comment]
> * Update circle ci file for nodejs 14

updated circle cli for node js 14

**Review by rajeshkumaravel — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: 2.1
 jobs:
   build:
     docker:
-      - image: circleci/node:12.13.0
```
Comment (swayangjit) [inline .circleci/config.yml:5]: Use - image: cimg/node:14.17.3   circleci/node:14.17.3 is deprecated

**Review by swayangjit — COMMENTED** [review]
Update the circle ci image

**Comment by princegupta1131** [comment]
> Update the circle ci image

updated image: cimg/node:14.17.3

File: projects/common-consumption/src/lib/card/group-card/group-card.component.scss [hunk 2]
```diff
@@ -1,6 +1,5 @@
-@import "../../../../../../node_modules/@project-sunbird/sb-styles/assets/mixins/mixins";
-@import "../../../../../../node_modules/@project-sunbird/sb-styles/assets/mixins/custom-mixins";
-@import "../../../../../../node_modules/@project-sunbird/sb-styles/assets/loadingshadows";
+@use "~@project-sunbird/sb-styles/assets/mixins/mixins" as *;
+@use "~@project-sunbird/sb-styles/assets/loadingshadows" as *;
```
Comment (rajeevsathish) [inline projects/common-consumption/src/lib/card/group-card/group-card.component.scss:2]: why do you need two @use ?

Comment (princegupta1131) [reply projects/common-consumption/src/lib/card/group-card/group-card.component.scss:2]: two @use -> both are different linkings
one is for mixin linking, another one styles linking

**Review by rajeevsathish — APPROVED** [review]

**Review by swayangjit — APPROVED** [review]

**Pull request merged**